pkg/subctl/cmd/root.go

```diff
@@ -112,16 +112,27 @@ func handleNodeLabels(config *rest.Config) error {
 			fmt.Printf("  - %s\n", node.GetName())
 		}
 	} else {
+		var node string;
 		answer, err := askForGatewayNode(clientset)
 		if err != nil {
 			return err
 		}
 		if answer.Node == "" {
 			fmt.Printf("* No worker node found to label as the gateway\n")
+			nodes, err := clientset.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+			if err != nil {
+				return err
+			}
+			if len(nodes.Items) == 0 {
+				return fmt.Errorf("There are no nodes in the cluster")
+			}
+			node = nodes.Items[0].GetName()
+			fmt.Printf("* Labelling %s as gateway\n", node)
```

Comment:
This seems unrelated to the broker namespace change, or am I missing something?

Reply:
Apologies, this should not be part of the commit. Is there an easy way to remove this from the PR, or should I submit a new PR?

Reply:
You can amend your commit and force push your branch; you’ll need to do that anyway because the additional commit generated by GitHub with my suggestions doesn’t carry a DCO... If you’d like help with the git commands feel free to ask.

Reply:
Thx, that worked.